contrib/firejail-welcome.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# This file is part of Firejail project
+# Copyright (C) 2014-2020 Firejail Authors
+# License GPL v2
+
+export LANG=en_US.UTF8
+
```

Comment:
zenity and sudo might not be installed. consider checking for them at the beginning and abort (and print an error message) if they are not available.